### PR TITLE
fix related submissions file names (WP-806)

### PR DESCRIPTION
**Pull request by vsolovei-smartling**
A cache related issue

### CODE REVIEW

File: inc/Smartling/Services/ContentRelationsDiscoveryService.php [hunk 1]
```diff
@@ -238,6 +238,12 @@ public function createSubmissions(UserTranslationRequest $request): void
         $job = $request->getJobInformation();
         $batchUid = $this->getBatchUid($profile, $job);
         $jobInfo = new JobEntityWithBatchUid($batchUid, $job->getName(), $job->getId(), $profile->getProjectId());
+
```
Comment (vsolovei-smartling) [inline inc/Smartling/Services/ContentRelationsDiscoveryService.php:241]: Refactoring, not fix-related

Comment (PavelLoparev) [reply inc/Smartling/Services/ContentRelationsDiscoveryService.php:241]: What does this change fix? Or what is this refactoring for?

**Review by PavelLoparev — CHANGES_REQUESTED** [review]

File: inc/Smartling/Helpers/ContentHelper.php [hunk 2]
```diff
@@ -145,7 +145,7 @@ private function getWrapper(string $contentType): EntityHandler
 
     public function readSourceContent(SubmissionEntity $submission): Entity
     {
-        if (false === ($cached = $this->getRuntimeCache()->get($submission->getId(), 'sourceContent'))) {
+        if (false === ($cached = $this->getRuntimeCache()->get($this->getCacheKey($submission), 'sourceContent'))) {
```
[the remark this answers is not in the document]
Comment (vsolovei-smartling) [reply inc/Smartling/Helpers/ContentHelper.php:148]: Done

File: inc/Smartling/Services/ContentRelationsDiscoveryService.php (hunk 1 above)
Comment (vsolovei-smartling) [reply inc/Smartling/Services/ContentRelationsDiscoveryService.php:241]: Early return on bulk submit screen to prevent potential unneeded actions

**Review by dimitrystd — APPROVED** [review]

**Pull request merged**